src/model_signing/_cli.py

```diff
@@ -236,7 +236,7 @@ def _sign_sigstore(
     Passing the `--use_staging` flag would use that instance instead of the
     production one.
     """
-    signer = sigstore.SigstoreDSSESigner(
+    signer = sigstore.SigstoreSigner(
```

Comment:
shall we de-dupe again?
```python
signer = sigstore.Signer(
```
```python
verifier = sigstore.Verifier(
```

Reply:
Or is there a downside for those that will import via
```python
from sigstore import SigstoreSigner
```

Reply:
Yup, planning to when merging with the key and cert signer.

Reply:
> Or is there a downside for those that will import via
> 
> ```python
> from sigstore import SigstoreSigner
> ```

It shouldn't, we recommend import module not symbol everywhere